chi/lease.py

```diff
@@ -565,6 +573,7 @@ def add_network_reservation(
     of_controller_ip=None,
     of_controller_port=None,
     vswitch_name=None,
+    resource_properties=[],
```

Comment:
The other month, Adam and Jason pointed out to me that having a list as a default argument. I think it's OK in this scenario since you are doing `.copy()` but we may want to avoid the scenario in entirety.

See this article for more: https://docs.python-guide.org/writing/gotchas/

Reply:
good to know! fixed!